languages/csharp/exercises/concept/switch-statements/.meta/design.md

```diff
@@ -0,0 +1,19 @@
+## Learning objectives
+
+- Know how to use simple `switch` statements.
+- Know how to include pattern matching on types.
+- Know how to include guards
+
+## Out of scope
+
+- switch expressions
+- pattern matching tuples
+
+## Concepts
+
+- `switch-statements`
+
+## Prerequisites
+
+- `enums`
+- `classes`
```

Comment:
Do we need to add the `inheritance` concept here too, as that explains that `object` is the base class of all classes/structs.

Reply:
done